File: 프로그래머스/lv0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.py
def solution(numbers):
    numbers.sort(reverse= True) # 큰수가 맨 앞으로 오게 정렬(sort)
    if numbers[0] * numbers [1] < numbers[-1] * numbers[-2]: # if문으로 -인 경우도 포함해서 계산
        dap = numbers[-1] * numbers[-2]
    else:
        dap = numbers[0] * numbers [1]
    return dap

File: 프로그래머스/lv0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.swift
import Foundation

func solution(_ numbers:[Int]) -> Int {
    var numberscount = numbers.count
    var numberssorted = numbers.sorted()
    var first = numberssorted[0] * numberssorted[1]
    var second = numberssorted[numberscount - 2] * numberssorted[numberscount - 1]
    return first > second ? first : second
}
